Makefile

```diff
@@ -0,0 +1,35 @@
+-include .env
+export
+
+# setup for docker-compose-ci build directory
+# delete "build" directory to update docker-compose-ci
+
+ifeq (,$(wildcard ./build/))
+    $(shell git submodule update --init --remote)
+endif
+
+EXTENSION=SemanticGlossary
+
+# docker images
+MW_VERSION?=1.39
+PHP_VERSION?=8.1
+DB_TYPE?=sqlite
+DB_IMAGE?=""
+
+# extensions
+SMW_VERSION?=4.1.3
+PF_VERSION ?= 5.5.1
```

Comment:
Does SG really need Page Forms and Semantic Forms Select? If not, just remove the versions to avoid unnecessary installations.

Reply:
You can always test locally by simply run a "make ci"

Reply:
You're right.